project/Component/NefRender/Errors/MarkdownError.swift

```diff
@@ -4,28 +4,28 @@ import Foundation
 
 public enum RenderError: Error {
     case renderContent
-    case render(page: URL)
-    case create(file: URL)
+    case renderPage(_ page: URL)
+    case renderPlayground(_ playground: URL)
+    case renderPlaygrounds
     case getPlaygrounds(folder: URL)
-    case getPages(folder: URL)
-    case structure
+    case getPages(playground: URL)
 }
 
 extension RenderError {
     var information: String {
         switch self {
         case .renderContent:
             return "can not render content"
-        case .render(let page):
+        case .renderPage(let page):
             return "can not render page '\(page.path)'"
-        case .create(let file):
-            return "can not create the file '\(file.path)'"
-        case .structure:
-            return "could not create project structure"
+        case .renderPlayground(let playground):
+            return "can not render playground '\(playground.path)'"
+        case .renderPlaygrounds:
+            return "can not render playgrounds"
```

Comment:
The render prefix feels a bit redundant if the type name is `RenderError`, but there are two cases that do not have it... Are we sure the last two cases belong here?

Reply:
Yep, I refactored the Markdown API name but I forget to do the same with the error :):)